index.html

```diff
@@ -223,36 +223,36 @@ <h2><i>Kiss And Tell</i></h2>
           </div><!--/row-->
         </div><!--/.col-xs-12.col-sm-9-->
 
-        <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="sidebar">
+        <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="float-none2 sidebar">
           <div class="list-group">
             <a href="#" class="list-group-item active">Singles ♭</a>
 
             <a href="https://www.youtube.com/watch?v=STO4-8vkG0U&list=RDSTO4-8vkG0U" class="list-group-item">Fetish</a>
-            <audio controls>
+            <audio controls class="audio">
```

Comment:
The `audio` class isn't working for smaller screen sizes. First, try to make the singles list visible for smaller screen sizes. Then we can make it responsive. Also, you might need to wrap the entire singles list into one `div` to make changes.